packages/components/src/components/editor/browser/browser.stories.tsx

```diff
@@ -37,6 +43,16 @@ const DefaultBrowser = ({ applyFn, applyBtn, initSearch }: DefaultBrowserProps)
   const funcs = useBrowser(funcData);
   const cms = useBrowser(cmsData);
 
+  const argumentInput = (value: string, onChange: (change: string) => void) => {
+    return <DialogBrowserInput value={value} onChange={onChange} />;
+  };
+  const condition = useConditionBuilder({
+    generateConditionString: GenerateConditionString,
+    typeOptions: TypeOptions,
+    logicalOperatorOptions: LogicalOperatorOptions,
+    argumentInput
+  });
```

Comment:
I think I would directly include the function instead of define it above

```suggestion
  const argumentInput = (value: string, onChange: (change: string) => void) => {
    return <DialogBrowserInput value={value} onChange={onChange} />;
  };
  const condition = useConditionBuilder({
    generateConditionString: GenerateConditionString,
    typeOptions: TypeOptions,
    logicalOperatorOptions: LogicalOperatorOptions,
    argumentInput: (value, onChange) => <DialogBrowserInput value={value} onChange={onChange} />
  });
```